### DISCUSSION TITLE
Woodcutting - Forestry Flowers event remove interact for wrong flowers

### DISCUSSION THREAD

**Discussion by BlueSun288** [runelite/runelite#16900] [Ideas]
In the Flowers event the flowers have improperly large hitboxes which leads to clicking on adjacent flowers when your cursor is distinctly on the correct flower. Removing the ability to interact on wrong flowers would prevent the hitbox problems from being an issue.

**Comment by Felanbird** [comment]
Don't people use "wrong" flowers for speed sake in certain patterns? I doubt they'd want their ability to interact taken away in that case.

**Reply by BlueSun288** [reply]
Are you thinking of the compost? The only thing you can do with the wrong flowers is get a 'wrong flower' message.

**Reply by Felanbird** [reply]
Yes xd
